detached-examples/multiuser-scenario/index.js

```diff
@@ -44,7 +43,9 @@ class User {
     }
 }
 
-const scenarios = new Map();
+const scenarios = global.scenarios || new Map();
+
+global.scenarios = scenarios;
```

Comment:
Can be simplified to `global.scenarios = global.scenarios || new Map()`

Reply:
Thanks.